akka-http-core/src/main/scala/akka/http/impl/engine/client/OutgoingConnectionBlueprint.scala

```diff
@@ -203,9 +219,25 @@ private[http] object OutgoingConnectionBlueprint {
         }
```

Comment:
This comment was placed after onUpstreamFailure in the substreamHandler (at least on my UI :-) ).